lib/result-view.coffee

```diff
@@ -227,12 +226,9 @@ transformimeJupyter.consoleTextTransform.mimetype = [
 ]
 
 transform = transformime.createTransform [
-    transformimeJupyter.PDFTransform,
     transformime.ImageTransformer,
     transformimeJupyter.SVGTransform,
     transformimeJupyter.consoleTextTransform,
-    transformimeJupyter.LaTeXTransform,
     MarkdownTransform,
     transformime.HTMLTransformer,
-    transformimeJupyter.ScriptTransform
```

Comment:
It's in my TODO list to remove the intermediate objects that Hydrogen creates for each kernel response. Once I do that `_parseDataMime` will go and we will rely fully on transformime.

Will we remember then to restore this transformer?